lua/octo/ui/writers.lua

```diff
@@ -379,7 +379,6 @@ function M.write_details(bufnr, issue, update)
         local strongest_review = utils.calculate_strongest_review_state(reviewers[name])
         local reviewer_vt = {
           { name, "OctoUser" },
-          { " " },
```

Comment:
This space between the user and the review icon is no longer needed now that all of the icons have enough space. The additional space makes the icon closer to the following user, rather than the user it represents.